Xamarin.Forms.Maps.iOS/MapRenderer.cs

```diff
@@ -246,7 +246,7 @@ protected override void OnElementChanged(ElementChangedEventArgs<View> e)
 					if (FormsMaps.IsiOs9OrNewer)
 					{
 						// See if we've got an MKMapView available in the pool; if so, use it
-						mapView = MapPool.Get();
+						//mapView = MapPool.Get();
```

Comment:
If you're just not going to do anything, remove the `if` statement.

Reply:
@hartez You added the cache a while back. I'm wondering if it ever worked. Was there a UITest for the cache? Do you think it's ok to pull the entire cache/pool?

Reply:
@kingces95 According to git, it was added in https://github.com/xamarin/Xamarin.Forms/pull/467. So there is a test (39489). If that test were to pass* with the pooling code commented out, then the underlying leak has probably been fixed and we could remove it.

Alternatively, if you can figure out where the maps leak is coming from and fix it, that'd be awesome.

* It doesn't.

Reply:
> Also had to disable the pool as the recycled map views failed to render.

When you say "failed to render", what do you mean? Big blank spot where the map should be? Blank map? Crash?

Reply:
Blank